spec/pivotal_to_trello/pivotal_wrapper_spec.rb

```diff
@@ -1,32 +1,27 @@
 require File.expand_path(File.dirname(File.dirname(__FILE__)) + '/spec_helper')
 
 describe 'PivotalWrapper' do
-  let(:wrapper) { PivotalToTrello::PivotalWrapper.new('token') }
+  let(:wrapper) { PivotalToTrello::PivotalWrapper.new(PT_USER_1[:token]) }
 
   context '#initialize' do
     it 'sets the pivotal token' do
-      expect(::PivotalTracker::Client).to receive(:token=).with('token')
-      PivotalToTrello::PivotalWrapper.new('token')
+      expect(TrackerApi::Client).to receive(:new).with(token: PT_USER_1[:token])
+      PivotalToTrello::PivotalWrapper.new(PT_USER_1[:token])
     end
   end
 
   context '#project_choices' do
     it 'returns a hash of pivotal projects' do
-      project = OpenStruct.new(id: 'id', name: 'My Project')
-      expect(::PivotalTracker::Project).to receive(:all).and_return([project])
-      expect(wrapper.project_choices).to eq('id' => 'My Project')
+      expect(wrapper.project_choices).to eq({1027488=>"My Sample Project", 1027492=>"My Sample Project"})
     end
   end
 
   context '#stories' do
     it 'returns a sorted array of pivotal stories' do
```

Comment:
It would be nice to call wrapper.project here as that's more in line with the api, but the method is private currently.